pkg/network/ebpf/c/tracer-maps.h

```diff
@@ -1,167 +0,0 @@
-#ifndef __TRACER_MAPS_H
-#define __TRACER_MAPS_H
-
-#include "tracer.h"
-#include "bpf_helpers.h"
-
-enum telemetry_counter{tcp_sent_miscounts, missed_tcp_close, udp_send_processed, udp_send_missed};
```

Comment:
Can this live in `tracer.h` since it's not a map?

Reply:
`tracer.h` is included by go code, so I thought that would only contain types needed by that code. I can move this to the new header you proposed in your earlier comment.